### ISSUE TITLE
problem in ´continuity´ when extracting ROIs

### ISSUE THREAD

**Issue by DavidMF123** [gcharvin/DetecDiv#12]
hi,
i´ve noticed in this last  batch i processed that the ROIs after extraction make sudden ´jumps´ into different ROIs (i assume within the same fov). i saw this happening before but just for a few timepoints and then it went back to the original ROI (i´ll recheck that again anyway, just in case it was worse than i thought originally), but now i´m seing that after a bit of back and forth, the ROI is being actually switced. within the same fov, the jump is happening at the same time position across different ROIs, but it´s happeing at different time points in each fov. 
there´s no error messages popping out, so maybe it´s a problem of my setup? im reading and saving the images in an external hard drive via USB, maybe the data transfer gets confused? i don´t think is that though, as the switched ROI is not centered as it is originally. for various examples i´ve seen, i doubt that the trap is the ´same one´, as sometimes it goes from a empty trap to a filled trap with a big cell inside already budded.. i´m attaching a couple of screenshot examples, just in case i didn´t explain myself enough (although i think in this case the cell is the same, but im not sure). we´d appreaciate any input on how to deal with this
thx
![41-1](https://user-images.githubusercontent.com/66879859/194774772-94abf8de-cff3-4891-8f68-7cb65c817f1d.png)

![41-2](https://user-images.githubusercontent.com/66879859/194774759-154ed8ac-e4ed-42d4-a785-97bd3417c1cd.png)

**Comment by gcharvin** [comment]
Did you try to disable the drift correction ? Do you still get this error ?

**Comment by DavidMF123** [comment]
indeed, by disabling drift correction the error went away. but with drift
correction on, i got it sometimes (and almost in every roi the last run
before i opened this ticket)
david

El dj., 8 de des. 2022, 16:40, Gilles Charvin ***@***.***> va
escriure:

**Comment by gcharvin** [comment]
OK. If it does not drift too much, I think you should keep going without drift correction (the network is somewhat robust with respect to XY translation), until I find a way to make the correction more reliable...

**Comment by DavidMF123** [comment]
yeah, luckily our movies are quite stable  usually. also, skipping drift
correction speeds up the process a lot. in the begining i was applying it
because i felt it was 'more correct' but it's not necessary most of the
times actually
david

El dj., 8 de des. 2022, 16:56, Gilles Charvin ***@***.***> va
escriure:

**Comment by gcharvin** [comment]
I agree, it s quite useless in the end because objects don t need to be tracked over time anyway...
